src/com/AbstractFactory/AbstractFactory.java

```diff
@@ -0,0 +1,27 @@
+package com.AbstractFactory;
+
+public class AbstractFactory {
+	
+	public SQLFactory getConnection(String acc) {
+		if(acc.equalsIgnoreCase("JDBC"))
```

Comment:
a good practice is to include brackets even if not completely necessary to improve readability